Source/controls/plrctrls.cpp

```diff
@@ -1096,26 +1075,24 @@ void WalkInDir(int playerId, AxisDirection dir)
 {
 	auto &player = plr[playerId];
 
-	const int x = player.position.future.x;
-	const int y = player.position.future.y;
+	const auto future{ player.position.future };
 
 	if (dir.x == AxisDirectionX_NONE && dir.y == AxisDirectionY_NONE) {
 		if (sgbControllerActive && player.walkpath[0] != WALK_NONE && player.destAction == ACTION_NONE)
-			NetSendCmdLoc(playerId, true, CMD_WALKXY, { x, y }); // Stop walking
+			NetSendCmdLoc(playerId, true, CMD_WALKXY, future); // Stop walking
 		return;
 	}
 
 	const Direction pdir = FaceDir[static_cast<std::size_t>(dir.x)][static_cast<std::size_t>(dir.y)];
-	const int dx = x + Offsets[pdir][0];
-	const int dy = y + Offsets[pdir][1];
+	const auto delta{ future + pdir };

```

Comment:
Isn't the style normally:
```suggestion
	const auto delta = future + pdir;
```